pkg/controller/migmigration/annotation.go

```diff
@@ -95,32 +95,53 @@ func (t *Task) annotateStageResources() error {
 	if err != nil {
 		return liberr.Wrap(err)
 	}
+	itemsUpdate := 0
```

Comment:
Assume this var name is supposed to be `itemsUpdated`

Reply:
I missed that, changing the name of the variable

Reply:
```suggestion
	itemsUpdated := 0
```